backend/controller/ml_model/templates/output.sql

```diff
@@ -0,0 +1,95 @@
+CREATE OR REPLACE TABLE `{{project_id}}.{{model_dataset}}.output` AS (
+  WITH prior_table AS (
+    SELECT SPLIT(MAX(table_id), 'events_')[OFFSET(1)] AS suffix
+    FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`
+    WHERE table_id LIKE 'events_%'
+    AND table_id NOT IN (
+      SELECT MAX(table_id) FROM `{{project_id}}.{{ga4_dataset}}.__TABLES_SUMMARY__`)
+  ),
+  events AS (
+    SELECT DISTINCT
+      {{unique_id}},
+      event_name AS name
+      event_params AS params
+    FROM `{{project_id}}.{{ga4_dataset}}.events_*`
+    WHERE _TABLE_SUFFIX = (SELECT suffix FROM prior_table)
+  ),
+  users_with_score AS (
+    SELECT DISTINCT
+      {{unique_id}}
+    FROM events, UNNEST(params) AS params
+    WHERE name = 'prop_score'
+    AND params.value.string_value = 'Predicted_Value'
+  ),
+  users_without_score AS (
+    SELECT DISTINCT
+      {{unique_id}}
+    FROM events
+    WHERE {{unique_id}} NOT IN (
+      SELECT {{unique_id}} FROM users_with_score)
+  ),
+  {% if label.is_score %}
+  prepared_predictions AS (
+    SELECT DISTINCT
+      {% if unique_id == 'user_id' %}
+      p.user_id,
+      {% endif %}
+      p.user_pseudo_id AS client_id,
+      {% if label.is_conversion %}
```

Comment:
If label.is_score and label.is_conversion are mutually exclusive, then no need to check it, right?  If they are mutually exclusive, then the 'else' statement would always be executed, so no need for the 'if' statement.

Reply:
The label can be output as a score without necessarily being output as a conversion. So, label.is_score and label.is_conversion can both be true or is_score could be true while is_conversion is false. When you specify in the UI that the output is a score it gives you the option to specify whether or not you'd like the output value to be a conversion instead of a score (while the score is also included as an additional value).

Perhaps the verbiage is confusing here. Happy to change this if you know of a better way to encapsulate this idea as I'm not certain what I would call this variable.